Exiled.Events/EventArgs/Scp079/RoomBlackoutEventArgs.cs

```diff
@@ -0,0 +1,83 @@
+// -----------------------------------------------------------------------
+// <copyright file="RoomBlackoutEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Scp079
+{
+    using Exiled.API.Features;
+    using Exiled.API.Features.Roles;
+    using Exiled.Events.EventArgs.Interfaces;
+
+    using MapGeneration;
+    using PlayerRoles.PlayableScps.Scp079;
+
+    /// <summary>
+    ///     Contains all information before SCP-079 blacks out the lights in a room.
+    /// </summary>
+    public class RoomBlackoutEventArgs : IPlayerEvent, IRoomEvent, IDeniableEvent
+    {
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="RoomBlackoutEventArgs" /> class.
+        /// </summary>
+        /// <param name="player">
+        ///     <inheritdoc cref="Player" />
+        /// </param>
+        /// <param name="roomIdentifier">
+        ///     <inheritdoc cref="Room" />
+        /// </param>
+        /// <param name="blackoutduration">
+        ///     <inheritdoc cref="BlackoutDuration" />
+        /// </param>
+        /// <param name="auxiliaryPowerCost">
+        ///     <inheritdoc cref="AuxiliaryPowerCost" />
+        /// </param>
+        /// <param name="cooldown">
+        ///     <inheritdoc cref="Cooldown" />
+        /// </param>
+        /// <param name="isAllowed">
+        ///     <inheritdoc cref="IsAllowed" />
+        /// </param>
+        public RoomBlackoutEventArgs(Player player, RoomIdentifier roomIdentifier, float auxiliaryPowerCost, float blackoutduration, float cooldown, bool isAllowed)
+        {
+            Player = player;
```

Comment:
We should do like in the event above: `Player = player.GetHub(hub)`.

Reply:
Done